src/components/views/settings/Notifications.js

```diff
@@ -15,33 +15,33 @@ limitations under the License.
 */
 
 'use strict';
-var React = require('react');
+const React = require('react');
```

Comment:
when updating to ES6, the correct way to fix these imports is actually to turn them into:
```
import React from 'react';
```
This technically works, but is very much not best practice.